datamodels/2.x/authent-cas/dictionaries/it.dict.authent-cas.php

```diff
@@ -6,7 +6,7 @@
  * @license     http://opensource.org/licenses/AGPL-3.0
  */
 Dict::Add('IT IT', 'Italian', 'Italiano', array(
-	'CAS:Error:UserNotAllowed' => 'User not allowed~~',
-	'CAS:Login:SignIn' => 'Sign in with CAS~~',
-	'CAS:Login:SignInTooltip' => 'Click here to authenticate yourself with the CAS server~~',
+	'CAS:Error:UserNotAllowed' => 'Utente non autorizzato~~',
+	'CAS:Login:SignIn' => 'Accedi con CAS~~',
+	'CAS:Login:SignInTooltip' => 'Clicca qui per autenticarti con il server CAS~~',
```

Comment:
```suggestion
	'CAS:Error:UserNotAllowed' => 'Utente non autorizzato',
	'CAS:Login:SignIn' => 'Accedi con CAS',
	'CAS:Login:SignInTooltip' => 'Clicca qui per autenticarti con il server CAS',
```

Example of entries that should no longer have trailing `~~`